profile/repositories-list.md

```diff
@@ -0,0 +1,147 @@
+|---------------|-------------------------------------------------------------------------------|
+| [mDoc Document Storage (Android)](https://github.com/eu-digital-identity-wallet/eudi-lib-android-wallet-document-manager)    | This library provides a set of classes to manage documents and their cryptographic keys in an EUDI Android Wallet. |
+
+
+### Revocation Libraries
+
+| Repository          | Description                                                                   |
+|---------------|-------------------------------------------------------------------------------|
+| [Token Status List (Kotlin)](https://github.com/eu-digital-identity-wallet/eudi-lib-kmp-statium) |  Statium is a Kotlin multiplatform library supporting JVM and Android platforms. It implements the Token Status List Specification draft 10, and allows callers to check the status of a "Referenced Token" as defined in the specification, effectively enabling applications to verify if tokens are valid, revoked, or in other states. |
+| [Token Status List (Swift)](https://github.com/eu-digital-identity-wallet/eudi-lib-ios-statium-swift) |  Statium is a Swift library for iOS and macOS. It implements the Token Status List Specification draft 10, and allows callers to check the status of a "Referenced Token" as defined in the specification, effectively enabling applications to verify if tokens are valid, revoked, or in other states.|
+
+
+### rQES: Low Level Libraries
+
+| Repository          | Description                                                                   |
+|---------------|-------------------------------------------------------------------------------|
+| [EUDI rQES CSC library (Kotlin)](https://github.com/eu-digital-identity-wallet/eudi-lib-jvm-rqes-csc-kt) |  This is a Kotlin library targeting JVM, that supports the Cloud Signature Consortium API (version 2) protocol. |
+| [EUDI rQES CSC library (Swift)](https://github.com/eu-digital-identity-wallet/eudi-lib-ios-rqes-csc-swift) |  This is a Swift library targeting JVM, that supports the Cloud Signature Consortium API (version 2) protocol. |
+
+### rQES: Core and Kit Libraries
+
+| Repository          | Description                                                                   |
+|---------------|-------------------------------------------------------------------------------|
+| [EUDI Remote Qualified Electronic Signature (RQES) Core library for Android](https://github.com/eu-digital-identity-wallet/eudi-lib-android-rqes-core) |  This module provides the core functionality for the EUDI Wallet for Android, focusing on the Remote Qualified Electronic Signature (RQES) service. |
+| [EUDI Remote Qualified Electronic Signature (RQES) Core library for iOS](https://github.com/eu-digital-identity-wallet/eudi-lib-ios-rqes-kit) |  This module provides the core functionality for the EUDI Wallet for iOS, focusing on the Remote Qualified Electronic Signature (RQES) service. |
```

Comment:
iOS repo is marked as `-kit` while Android` as `-core` (in their urls) 

is this correct?